airgun/views/fact.py

```diff
@@ -13,6 +13,7 @@ class HostFactView(BaseLoggedInView, SearchableViewMixinPF4):
             'Name': Text("./a"),
         },
     )
+    expand_value = Text("//div/a[contains(span/@class, 'glyphicon-plus')]")
```

Comment:
I think this could confuse later, so can we rename this field?
```suggestion
    expand_fact_value = Text("//div/a[contains(span/@class, 'glyphicon-plus')]")
```